packages/nodejs/deps-v8-BUILD.gn.patch

```diff
@@ -0,0 +1,11 @@
+--- ./deps/v8/BUILD.gn	2021-06-03 07:15:30.000000000 +0530
++++ ./deps/v8/BUILD.gn.mod	2021-07-17 17:32:54.134091233 +0530
+@@ -17,6 +17,8 @@
+ import("gni/snapshot_toolchain.gni")
+ import("gni/v8.gni")
+ 
++v8_target_cpu = getenv("DEST_CPU")
```

Comment:
Also set `target_cpu` to `getenv("DEST_CPU")` and try compiling for disabled arche after current build passes/fails

Reply:
I am currently trying with a different approach... Willl try it if that fails

Reply:
Well that approach won't work probably. 
See https://v8.dev/docs/cross-compile-arm

In the docs for cross compiling, all the code blocks have `v8_target_cpu` and `target_cpu` set to same value

Reply:
Done... Let's see if it works...